### PR TITLE
Weld types UI and renaming vars

### PR DESCRIPTION
**Pull request by emagdalenaC2i**
I think this could be the UI ;-)

### CODE REVIEW

**Comment by mbartlett21** [comment]
@emagdalenaC2i 
You can merge this if you want

File: weld.fs.js [hunk 1]
```diff
@@ -142,29 +151,44 @@ export const weld = defineFeature(function(context is Context, id is Id, definit
...
+            definition.buttRootGap is boolean;
+
+            if (definition.buttRootGap)
+            {
+                annotation { "Name" : "Root width", "UIHint" : "REMEMBER_PREVIOUS_VALUE" }
+                isLength(definition.buttRootGapWidth, SHELL_OFFSET_BOUNDS);
+
+                annotation { "Name" : "Root height", "UIHint" : "REMEMBER_PREVIOUS_VALUE" }
+                isLength(definition.buttRootGapHeight, SHELL_OFFSET_BOUNDS);
+            }
```
Comment (mbartlett21) [inline weld.fs.js]: This section for root gap should be moved up and put in an `if` like this:
```
if(definition.weldType != WeldType.SQUARE_BUTT_WELD){
            annotation { "Name" : "Root Gap", "UIHint" : "REMEMBER_PREVIOUS_VALUE", "Default" : true }
            definition.buttRootGap is boolean;
             if (definition.buttRootGap)
            {
                annotation { "Name" : "Root width", "UIHint" : "REMEMBER_PREVIOUS_VALUE" }
                isLength(definition.buttRootGapWidth, SHELL_OFFSET_BOUNDS);
                 annotation { "Name" : "Root height", "UIHint" : "REMEMBER_PREVIOUS_VALUE" }
                isLength(definition.buttRootGapHeight, SHELL_OFFSET_BOUNDS);
            }
}
```

Comment (mbartlett21) [reply weld.fs.js]: Ok
You should still move it up, just without the if.
Currently it will show with a fillet weld

**Comment by emagdalenaC2i** [comment]
@mbartlett21 Please, check and merge this PR

**Pull request merged**